recipes/models.py

```diff
@@ -0,0 +1,12 @@
+from django.db import models
+
+
+# Create your models here.
+
+
+class Recipe(models.Model):
+    name = models.CharField(max_length=100)
+    description = models.CharField(max_length=255)
```

Comment:
Perhaps this field can be optional, in case you don't want to make the description mandatory. In that case you can play with `null=True`
https://docs.djangoproject.com/en/4.0/ref/models/fields/#null